.travis.yml

```diff
@@ -15,50 +37,57 @@ env:
     - xTHREADING=1 xMPI=0 xGSL=0 xLTDL=1 xREADLINE=1 xPYTHON=0 xMUSIC=0 xSTATIC_ANALYSIS=1 CACHE_NAME=JOB   # only threading
     - xTHREADING=1 xMPI=1 xGSL=1 xLTDL=1 xREADLINE=1 xPYTHON=1 xMUSIC=1 xSTATIC_ANALYSIS=1 CACHE_NAME=JOB   # full
 matrix:
-  # do notify immediately about it when a job of a build fails.
+  # Do notify immediately about it when a job of a build fails.
   fast_finish: true
 cache:
   directory:
     - $HOME/.cache
 
 before_install:
+  - echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
+  - echo "+         P R E P A R E   N E S T   B U I L D   E N V I R O N M E N T         +"
+  - echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
   - export SOURCEDIR=$PWD
   - cd $HOME/build
   - echo $PATH
 
-  # update package repository status (-qq is more quiet)
+  # Update package repository status (-qq is more quiet).
   - sudo rm -rf /var/lib/apt/lists/*
   - ls /etc/apt/sources.list.d/
   - sudo apt-get update -qq
 
-  # remove any obsolete libraries
+  # Remove any obsolete libraries.
   - sudo apt-get autoremove
 
-  # from installation webpage http://www.nest-simulator.org/installation-2/#Standard_configuration
+  # Install packages required by NEST.
+  # (Please see also: http://www.nest-simulator.org/installation-2/#Standard_configuration )
   - sudo apt-get install -y build-essential cmake libltdl7-dev libreadline6-dev libncurses5-dev libgsl0-dev python-all-dev python-numpy python-scipy python-matplotlib ipython
 
-  # Install pip and cython
+  # Install pip and Cython.
   - wget https://bootstrap.pypa.io/get-pip.py
   - sudo python get-pip.py
   - sudo pip install -U setuptools
   - pip install --user cython==0.23.4
 
-  # MPI
+  # Install package 'terminaltables' required by 'parse_travis_log.py' to create the build summary.
```

Comment:
If you use quotes around the packages here, quotes probably should also be used for command and packages everywhere else in the file. Alternatively remove them here, as the meaning is also clear without them.